x-pack/plugins/security/server/authentication/authenticator.test.ts

```diff
@@ -1872,6 +1872,25 @@ describe('Authenticator', () => {
       expect(mockOptions.session.invalidate).not.toHaveBeenCalled();
     });
 
+    it('if session does not exist and providers is empty, returns whatever authentication provider returns.', async () => {
```

Comment:
__note:__ it seems we have the same issue as in `authentication_service.test.ts` here as well. The default config still has non-empty `sortedProviders` and hence the test doesn't fail even without the fix.